src/Symfony/Component/Serializer/Normalizer/GetSetMethodNormalizer.php

```diff
@@ -108,13 +108,13 @@ public function denormalize($data, $class, $format = null, array $context = arra
                 $attribute = $this->nameConverter->denormalize($attribute);
             }
 
-            $allowed = false === $allowedAttributes || in_array($attribute, $allowedAttributes);
-            $ignored = in_array($attribute, $this->ignoredAttributes);
+            $allowed = false === $allowedAttributes || \in_array($attribute, $allowedAttributes, true);
```

Comment:
Could setting the `strict` argument to `true` create some BC break in some obscure edge case?

Reply:
I agree, let's remove all "true" added to in_array checks (the 3rd arg is not needed to benefit from the optim)

Reply:
Removed, but both are strings so it cannot have a BC break.